src/lib/components/atoms/SpokenText.svelte

```diff
@@ -5,6 +5,7 @@
 	import { AudioIcon } from '$icons'
 
 	export let text = ''
+	export let medium: boolean = false
```

Comment:
Change this to make it possible to supply a font size with a prop like `--font-size`. Just like we do now with `--align`.